### ISSUE TITLE
Find workaround for AWACS

### ISSUE THREAD

**Issue by lukrop** [lukrop/GCICAP#3]
AWACS aircraft seem to not work with `isTargetDetected` and therefor don't report any enemy flights, eliminating them as useful "radar resource".

Check if it's possible to hack some simple detection logic or wait for upstream (ED) fix.

**Comment by cjbehm** [comment]
I think you just have to use the unit controller instead of group controller. I had a test script a little while back and I just revisited. This seems to work fine

Controller.isTargetDetected(awacsUnit,possibleIntruder,Controller.Detection.RADAR)

The notes at Hoggit wiki suggest it behaves differently between ground and air groups (http://wiki.hoggit.us/view/DCS_func_isTargetDetected)

**Comment by cjbehm** [comment]
Yup, got this working with a really small set of changes. Not sure how I could submit this set of changes separate from the engagement changes (my PR experience being limited) so I can just hold this until those are done.

**Comment by lukrop** [comment]
Great!
You could checkout commit b10cb11090585ee1dc3665b59a4b0d13b07c60d9 and create a new branch, apply your changes and create a new PR for this branch.

If needed you could stash your changes.

**Comment by cjbehm** [comment]
Yeah, I think I see how I can get that in without creating a merge conflict while we make sure the larger engagement PR gets careful review.

**Comment by cjbehm** [comment]
#23 Here it is, short and sweet :)

**Comment by lukrop** [comment]
Great! Looking forward to closing this issue!
